homeassistant/components/unifi/device_tracker.py

```diff
@@ -150,16 +154,31 @@ def __init__(self, client, controller):
         """Set up tracked client."""
         super().__init__(client, controller)
 
-        self.heartbeat_check = False
         self._controller_connection_state_changed = False
 
-        self._last_seen = client.last_seen or 0
```

Comment:
Can `self._last_seen` be removed?

Reply:
Yes the use of it was removed in a previous PR